stage1/gc/gc.go

```diff
@@ -85,3 +90,14 @@ func gcNetworking(podID *types.UUID) error {
 
 	return nil
 }
+
+func removeJournalLink(uuid *types.UUID) error {
+	machineID := strings.Replace(uuid.String(), "-", "", -1)
+	journalLink := filepath.Join("/var/log/journal", machineID)
```

Comment:
The name in `/etc/machine-id` is not the name we use to register the container with machined. The former is the pod UUID without dashes and the latter is `rkt-${POD_UUID}`.

I'm pretty confident we'll keep the pod UUID without dashes as a machine id.